src/main.rs

```diff
@@ -1,3 +1,5 @@
+extern crate core;
```

Comment:
Small question, why is core suddenly required ?

Reply:
I think that was an accidental import that then got missed by clippy.